packages/@functionless/aws-cognito-constructs/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@functionless/aws-cognito-constructs",
+  "version": "0.27.0",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib",
+    "docs"
```

Comment:
do we want to publish the docs to npm?

Reply:
I think so, yeah. So that developers can read the from in the IDE?

Reply:
hmm, never seen that, but sure!